.github/workflows/tests.yaml

```diff
@@ -1,12 +1,6 @@
 name: Tests
 
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
```

Comment:
You're right, it does not belong in this PR. I'll remove it.

FWIW I changed so that CI tests run my github fork of anchor, so I didn't have to wait for someone to authorize the CI run before I can see results.